pkg/static_unit/plugin_static_unit.go

```diff
@@ -98,6 +105,47 @@ type staticAssetPathMatcher struct {
 	err         error
 }
 
+// ModifyPathsForAnnotatedFile transforms the staticAssetPathMatcher sharedFiles and staticFiles to be absolute paths from the klotho project root, without the prefix '/'.
+// This is done to conform to the file path structure of input files.
+func (m *staticAssetPathMatcher) ModifyPathsForAnnotatedFile(path string) error {
+	newInclude := []string{}
+	for _, pattern := range m.staticFiles {
+		absPath, err := filepath.Abs(pattern)
+		if err != nil {
+			return err
+		}
+		if absPath == pattern {
```

Comment:
im not quite understanding why this is a special case. Won't every `absPath` have a `/`? Why is the prefix only trimmed if the `filpath.Abs(pattern)` seemingly does nothing to the path?

Reply:
so our paths for files are like `src/somefile.txt`

if they specify an absolute path of `/src/somefile.txt` we wont match unless we remove the prefix /

The other scenario after that is for relative paths, which wont start with / anyways.

The only real reason we check absPath == pattern is just to make sure its a true absolute path.

If that doesnt make sense we can rework this or have another discussion, going to push to get the search the deck stuff going for ala